src/TaxonomyTerm.php

```diff
@@ -63,33 +69,33 @@ class TaxonomyTerm extends DataObject implements PermissionProvider
 
     public function getCMSFields()
     {
-        $fields = parent::getCMSFields();
+        $this->beforeUpdateCMSFields(function (FieldList $fields) {
```

Comment:
Wrapping the implementation in `beforeUpdateCMSFields()` is best practice. This is partially unrelated refactoring, but it also is necessary to make sure the `Children` field is a gridfield before extension hooks fire, which will keep those functioning the same before and after this change.

Reply:
All other changes to this method (unless otherwise stated) are just explicitly using a `GridField` for the `Children` relation, instead of the auto-scaffolded searchable dropdown.